### DISCUSSION TITLE
Add support for Fowarded header (RFC 7239)

### DISCUSSION THREAD

**Discussion by nhairs** [Kludex/uvicorn#2236] [Ideas] [answered]
We should probably support the `Forwarded` header which was standardised in [RFC 7239](https://datatracker.ietf.org/doc/html/rfc7239).

One of the benefits of this header over `X-Forwarded-*` headers is that it supports including the port of the client rather than just the address.

See also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded

**Comment by Kludex** [comment, marked as answer]
- I've created https://github.com/encode/uvicorn/issues/2237 from this discussion.
